pkg/cmd/server/bootstrappolicy/infra_sa_policy.go

```diff
@@ -116,7 +116,7 @@ func init() {
 				// BuildControllerFactory.buildDeleteLW
 				{
 					Verbs:     sets.NewString("get", "list", "create", "delete"),
-					Resources: sets.NewString("pods"),
```

Comment:
nope, don't grant pods/bind by default. only this plugin being active enforces anything on the pod creation itself